homeassistant/components/binary_sensor/mqtt.py

```diff
@@ -29,8 +30,8 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_PAYLOAD_OFF, default=DEFAULT_PAYLOAD_OFF): cv.string,
     vol.Optional(CONF_PAYLOAD_ON, default=DEFAULT_PAYLOAD_ON): cv.string,
-    vol.Optional(CONF_SENSOR_CLASS, default=None):
-        vol.Any(vol.In(SENSOR_CLASSES), vol.SetTo(None)),
```

Comment:
So I'm not sure why the schema was set up like this before. Is there a specific reason we want to silently ignore invalid classes?